ui/raidboss/data/06-ew/raid/p8s.ts

```diff
@@ -941,6 +960,201 @@ const triggerSet: TriggerSet<Data> = {
+        if (
+          data.footfallsDirs[0] !== undefined &&
+          data.footfallsOrder[0] !== undefined &&
+          data.footfallsDirs[1] === undefined &&
+          data.footfallsOrder[1] === undefined
+        ) {
+          if (!validDirs.includes(data.footfallsDirs[0])) {
+            console.error(`Blazing Footfalls: Unexpected dirs, got ${data.footfallsDirs[0]}}`);
+            return;
+          }
+
+          return output.firstTrailblaze!({
+            dir: dirToCard[data.footfallsDirs[0]],
+            concept: data.footfallsConcept,
+          });
+        }
+      },
+      outputStrings: {
+        firstTrailblaze: {
+          en: '${dir} Black Line => ${concept}',
+        },
+        north: Outputs.north,
+        east: Outputs.east,
+        south: Outputs.south,
+        west: Outputs.west,
+      },
+    },
+    {
+      id: 'P8S Blazing Footfalls Trailblaze 2',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['7106', '7107'], source: 'Hephaistos', capture: false }),
+      condition: (data) => data.trailblazeCount === 1,
+      durationSeconds: 3.9, // Keep up until Trailblaze
+      infoText: (data, _matches, output) => {
+        if (data.footfallsDirs[1] !== undefined && data.footfallsOrder[1] !== undefined) {
+          // Check if have valid dirs
+          const validDirs = [0, 1, 2, 3];
+          if (!validDirs.includes(data.footfallsDirs[1])) {
+            console.error(`Blazing Footfalls Reminder: Unexpected dirs, got ${data.footfallsDirs[1]}`);
+            return;
+          }
+
+          const dirToCard: { [dir: number]: string } = {
+            0: output.north!(),
+            1: output.east!(),
+            2: output.south!(),
+            3: output.west!(),
+          };
+
+          return output.trailblaze!({
+            dir: dirToCard[data.footfallsDirs[1]],
+            action: output[data.footfallsOrder[1]]!(),
+          });
+        }
+      },
+      outputStrings: {
+        trailblaze: {
+          en: '${dir} Black Line => ${action}',
+        },
+        crush: {
+          en: 'Crush',
+        },
+        impact: Outputs.knockback,
+        north: Outputs.north,
+        east: Outputs.east,
+        south: Outputs.south,
+        west: Outputs.west,
+      },
+    },
+    {
+      id: 'P8S Blazing Footfalls Crush/Impact Reminder',
+      // Reminder after Trailblaze for Impact/Crush Movement
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['793E'], source: 'Hephaistos' }),
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime),
+      durationSeconds: 4,
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          trailblaze: {
+            en: '${dir}',
+          },
+          trailblazeKnockback: {
+            en: '${dir} Knockback',
+          },
+          trailblazeKnockbackTo: {
+            en: '${dir1} Knockback to ${dir2}',
```

Comment:
This is super nitpicky, but it sounds a little bit weird to my ears to say "south knockback to right" vs "south knockback right". Would you mind having a `trailblazeKnockback`, a `trailblazeKnockbackToDir`, and a `trailblazeKnockbackSide` output string here, so the left/right doesn't get the word "to" in there?

Feel free to disagree (maybe this is just something I should customize personally), but I also feel like the final knockback doesn't need a dir1 in it and could just say "Knockback Right" or "Knockback Left".  It should be pretty clear that you have just gotten knocked back to south, that the knockback is coming from south.

Reply:
>Would you mind having a trailblazeKnockback, a trailblazeKnockbackToDir, and a trailblazeKnockbackSide output string here, so the left/right doesn't get the word "to" in there?

I think I understand what you're asking for here.

It's likely not recoverable if the player is pushed back to the wrong side during the second trailblaze as they are very likely to have used their immunity for the earlier mechanic... although I haven't even tested that this knockback from the wall can be surecasted/armslengthed (It looks like it can?).

Reply:
I didn't even think you could use immunity on this.

Really my main point is that: "knockback to left" is awkward to my ears; it should be "knockback left" (preferable) or "knockback to the left" (too long, in my opinon).  So, maybe differentiate if impact is the final mechanic ("knockback left") vs the penultimate one ("W knockback to S") via different output strings like `trailblazeKnockbackSide` and `trailblazeKnockbackToDir`, respectively.  I guess you could drop the "to" on the "N/E/S/W" one, but that'd certainly sound awkward if you replaced N/E/S/W with marker names? Not sure.